### PR TITLE
RFC: removed 'aurae' from client & client-macros

### PR DESCRIPTION
**Pull request by zanshin**
https://github.com/aurae-runtime/aurae/issues/371

First attempt a contributing to an open source project.

### CODE REVIEW

**Comment by zanshin** [comment]
@cla-bot check

File: aurae-client/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [package]
```
[the remark this answers is not in the document]
Comment (zanshin) [reply aurae-client/Cargo.toml:1]: Whoops. I missed that. Thanks

**Review by dmah42 — APPROVED** [review]

**Comment by dmah42** [comment]
this looks good.. waiting for the CI bots to agree.

**Comment by dmah42** [comment]
`auraed/src/cells/cell_service/cell_service.rs:43:5` and others need some import change love.

File: aer/src/lib.rs [hunk 2]
```diff
@@ -66,7 +66,7 @@ pub mod runtime;
 #[macro_export]
 macro_rules! execute {
     ($call:path, $req:ident) => {{
-        let client = ::aurae_client::AuraeClient::default().await?;
+        let client = ::client::AuraeClient::default().await?;
```
Comment (dmah42) [inline aer/src/lib.rs:69]: should this also be Client instead of AuraeClient?

Comment (zanshin) [reply aer/src/lib.rs:69]: You mean ::client::Client::default().await?;

Comment (dmah42) [reply aer/src/lib.rs:69]: yes. I think it should.

sorry for the rounds of comments. every time you add more files I see more 😅

Comment (zanshin) [reply aer/src/lib.rs]: No worries. I’m having fun exploring the code base.

**Review by dmah42 — APPROVED** [review]

**Comment by dmah42** [comment]
thank you and well done!

**Pull request merged**